docs/concepts/vega-chain.md

```diff
@@ -83,63 +96,123 @@ Exceptions to automatic nomination:
 
-**Un-nominate towards the end of the epoch** 
+If nominated tokens are moved to a different Ethereum address, they are un-nominated immediately, (equivalent to ['un-nominate now'](/docs/concepts/vega-chain#un-nominate-now)) and rewards are forfeited for that epoch. 
 
-A participant can un-nominate towards the end of the current epoch, which means the stake is not used for the validator from the following epoch. 
+#### Un-nominate towards the end of the epoch
+A participant can un-nominate towards the end of the current epoch, which means the stake is not used for the validator from the following epoch. The participant, and their nominated validator, is entitled to the rewards from that epoch (unlike when un-nominating now). 
 
 The action is announced in the next available block of the same epoch, but the nominator keeps their nomination active until the last block of that epoch. At that point, the tokens are returned. The nominator cannot move those tokens before the epoch ends.
 
-**Un-nominate now**
-
+#### Un-nominate now
 A participant can choose to un-nominate at any time, and the action is executed immediately following the block it is announced in (within the same epoch). 
 
 The participant will not receive any rewards from the validator in that epoch. The tokens are marked as available to the participant.
 
-### Staking rewards
-Validators and nominators both receive incentives from the network, depending on factors including how much stake is nominated. 
+## Staking rewards & penalties
 
-**To be considered for staking rewards, a participant must associate VEGA to a Vega key and nominate one or more validators.**
+### Rewards
+Validators and nominators both receive incentives for securing the network. The amount of those incentives, rewarded as VEGA, depends on factors including how much stake is nominated. 
 
-For restricted mainnet, rewards are distributed among validators in proportion to their total stake. The total stake includes a validator's own stake and the tokens nominated to that validator. 
+**To be considered for staking rewards, a tokenholder must associate VEGA to a Vega key and nominate one or more validators.**
 
-At the end of each epoch, reward payments are calculated per active validator, and then some of that reward is divided between their nominators. The proportion that goes to nominators is defined by the network parameter `reward.staking.delegation.delegatorShare`. 
+In each epoch, rewards are distributed among validators in proportion to the number of tokens they represent (i.e., their total stake). The total stake includes a validator's own stake and the tokens nominated to that validator. Of this reward, a fixed amount is distributed among the tokenholders the validator represents. Currently that is 88.3%, though validators can vote to change that value.
```

Comment:
the last statement is a bit inaccurate. this 0.883 is applied to the proportion of delegation by delegators, not on the reward amount that goes to the validator and its delegators. 

Lets say for example that validator 1 is due to get 1000 tokens for them and their delegators. 
And lets say that they have self stake of 3000 and delegation by delegators of 7000. 
That means that the delegators of validator1 will get 0.883 * 1000 * 7000/10000 = 618 which will be divided between the delegators to validator 1 based on their relative proportion of nomination in the validator. 
Validator 1 would get the rest 1000-618 = 382 tokens.

Reply:
I've cut that sentence for now but will reflect this better in another PR, thanks for explaining it!